storm-core/src/clj/org/apache/storm/ui/core.clj

```diff
@@ -134,12 +134,12 @@
 
 (defn logviewer-link [host fname secure?]
   (if (and secure? (*STORM-CONF* LOGVIEWER-HTTPS-PORT))
-    (UIHelpers/urlFormat "https://%s:%s/log?file=%s"
+    (UIHelpers/urlFormat "https://%s:%s/api/v1/log?file=%s"
```

Comment:
Yes agreed. There're some responses returning HTML page instead of JSON, so we may need to change it first and try to document it.